templates/lightning/generic.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE HTML>
```

Comment:
this file not used?

Reply:
i thought i remove this. having a look again

Reply:
nope, missed it. ill delete this line after the push from Alex